composer.json

```
@@ -10,7 +10,15 @@
 	"scripts": {
 		"lint": "./vendor/bin/phpcs --standard=phpcs.xml"
 	},
+	"autoload": {
+		"classmap": [
+			"./inc"
+		]
```

How do you feel about using the psr-4 standard instead? This would let the publisher use their namespace as the root namespace and forces users into a file/folder structure that matches the class namespaces. Admittedly this is a personal preference of mine so I'm going to push too hard here :)
```suggestion
		"psr-4": {
		    "PublisherName\\": "inc/"
		    }
```

---

I think the main reason I used classmap was because of the Modules thing. Then every module would have to have `Modules` in their namespaces..

But maybe this could be avoided by also adding the modules folder there in the settings? and not just inc?

---

Not sure what I did wrong, but I could not get this to work. Not even the `Module_Loader` class was being found... lmk if you want to give this a shot

---

I'll take a look at this today and we can get it merged